src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -101,6 +107,13 @@ class AuthScreens extends React.Component {
             User.getBetas();
         }, 1000 * 60 * 30);
 
+        // Update the users timezone if they have it set to automatic
+        const timezone = lodashGet(this.props.myPersonalDetails, 'timezone', CONST.DEFAULT_TIME_ZONE);
+        if (_.isObject(timezone) && timezone.automatic) {
+            timezone.selected = moment.tz.guess();
```

Comment:
A couple things about .guess(), it looks like it is not supported in all browsers (though it is supported by almost all latest versions), does there need to be a fallback / what would it return?

Additionally, it looks like it will always cache the detected timezone, I think that should be okay but is there ever a time we should ignore the cache?

Reply:
For browser compatibility/fallbacks, since we already use `moment.tz.guess()` in E.cash (and E.com) I think it's fine to continue using it as is.

Good point on the cache, I think we should be ignoring it in this case.